src/test/py/bazel/bazel_server_mode_test.py

```diff
@@ -19,14 +19,24 @@
 class BazelServerModeTest(test_base.TestBase):
 
   def testBazelServerMode(self):
-    self.ScratchFile('WORKSPACE')
+    self.ScratchFile('MODULE.bazel')
 
     _, stdout, _ = self.RunBazel(['info', 'server_pid'])
     pid1 = stdout[0]
     _, stdout, _ = self.RunBazel(['info', 'server_pid'])
     pid2 = stdout[0]
     self.assertEqual(pid1, pid2)
 
+  def testBazelServerModeStressTest(self):
```

Comment:
I can't reproduce the previous failure with this test on my local Linux machine.

Reply:
Hmm, looks like even the existing test case should have caught this.

Reply:
I'll verify and fix this later if necessary.

Reply:
testBazelServerModeStressTest did catch the failure on my linux workstation. The first test case only somethings fails. But anyway this PR should be good enough, thanks!